mail-client/mutt-wizard/mutt-wizard-20200430.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils
+
+DESCRIPTION="A system for automatically configuring mutt and isync"
+HOMEPAGE="https://github.com/LukeSmithxyz/mutt-wizard"
+
+COMMIT=6fe26400957d9163a75f60fa0c18f266f25fbc85
+SRC_URI="https://github.com/LukeSmithxyz/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE=""
+
+RDEPEND="
+	mail-client/neomutt:=[notmuch]
+	net-mail/isync:=[ssl]
+	mail-mta/msmtp
+	app-admin/pass
+"
+
+# needed because there is no 'all' target defined in MAKEFILE
+src_compile() {
+	return 0;
+}
+
+pkg_postinst() {
```

Comment:
Maybe only show this on new installs (check ${REPLACING_VERSIONS})?

Reply:
might be good to keep it for everyone, as it changes a fair over time, by adding a lot of features.